dashboard/test/controllers/reports_controller_test.rb

```diff
@@ -296,7 +296,7 @@ class ReportsControllerTest < ActionController::TestCase
 
     assert_response :success
 
-    assert flash[:error]
+    assert_equal "I can't find that user", css('.container .alert-danger')[0].text.strip
```

Comment:
does
`assert_select '.container .alert-danger', "I can't find that user"` not work?